### DISCUSSION TITLE
Jquery upgrade from 3.6.0 to 3.7.1

### DISCUSSION THREAD

**Discussion by lingesh-k-7833** [jquery/jquery#5367] [Q&A] [answered]
We are seeking your guidance regarding the upgrade procedure from version 3.6.0 to version 3.7.1. What should be noted before and after the upgrade?

**Comment by mgol** [comment, marked as answer]
This is a minor update so there are no breaking changes. You may read the blog posts for the summary of the changes and it’s always worth testing but we don’t expect breaking changes.
